server/server.js

```diff
@@ -49,7 +50,7 @@ app.use((req, res) => {
       RoutingContext(renderProps)
     );
 
-    const initialState = store.getState();
+    const initialState = req.params.token ? store.getState() : {}; // get token from request
```

Comment:
Not sure how send the session in the http request yet...especially on page refresh